.taskcluster.yml

```diff
@@ -0,0 +1,30 @@
+version: 0
+metadata:
+  name: "taskcluster-lib-monitor test"
+  description: "tests for metrics and monitoring library"
+  owner: "{{ event.head.user.email }}"
+  source: "{{ event.head.repo.url }}"
+
+tasks:
+  - provisionerId: "{{ taskcluster.docker.provisionerId }}"
+  workerType: "{{ taskcluster.docker.workerType }}"
+  extra:
+    github:
+      env: true
+      events:
+        - pull_request.opened
+        - pull_request.synchronize
+        - pull_request.reopened
+        - push
+  payload:
+    maxRunTime: 3600
+    image: "quay.io/mrrrgn/ubuntu-ci:0.0.1"
+    command:
```

Comment:
And similarly this command will need to be updated.

This command and the `image` associated with it for the above comment is best taken from https://github.com/taskcluster/taskcluster-github/blob/master/.taskcluster.yml#L24-L28

This is just due to bit-rot on the part of the `quay.io/mrrrgn` image. The fact that the command in this version is so much longer is annoying, but perhaps we can figure out a way to fix it soon!